### ISSUE TITLE
Document how to include the file processed by PathRewriterPlugin

### ISSUE THREAD

**Issue by AndrewRayCode** [skozin/webpack-path-rewriter#5]
Right now it is not clear how `PathRewriterPlugin` is supposed to access the files it touches. Are you supposed to `require()` the files? Are you supposed to add them as an entry point?

**Comment by skozin** [comment]
Yeah, this really needs documenting. I'll leave this issue opened until it is done. For now, please see [this](https://github.com/skozin/webpack-path-rewriter/tree/master/_example) for an example of basic usage (I've added it just now).
